### DISCUSSION TITLE
Finding context in Local Search

### DISCUSSION THREAD

**Discussion by therealcyberlord** [microsoft/graphrag#1019] [Q&A]
Is there a way to view the retrieved context alone in the local search? I’d like to verify the information retrieved and see the final prompt that the LLM receives.

I checked similar discussions but didn't find a solution—please let me know if I missed anything.

Thanks

**Comment by AlonsoGuevara** [comment]
Hi @therealcyberlord 
We just added this in our latest release 0.3.2

Here's the related PR: https://github.com/microsoft/graphrag/pull/1003

**Reply by loiccordone** [reply]
Hello!
Thank you for the PR, but how can I access `context_text `when using the CLI? It seems that only the response is returned with the command
`python -m graphrag.query --root . --method global "prompt"`
